rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -349,269 +210,29 @@ def get_stability_pool_positions(
             }
         return data
 
-    def liquity_staking_balances(
+    def get_stability_pool_positions(
```

Comment:
This one is called once in the rest.py file and the other in two places. I made them as a wrapped because they hide the internal logic of a base method and by their name is easy to understand what is being queried and its usage. You don't need to know anything of the underlying implementation

Reply:
if it's called in multiple places it's sort of okay. If not I think it's quite redundant. Remember, function calls are not free. Though probably this is not an optimizaiton we need to focus on. I will give you free hand on this

Reply:
I have decided to keep the methods as they are because:

1. They are called in different places
2. The abstract a more complex underlying logic that doesn't need to be known
3. Is code that is not execute frequently unless the user is constantly triggering it
4. Doesn't load always, only when querying balances if the module is active and when entering the staking page